src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -112,6 +112,7 @@ public static class ScriptConstants
         public const string FeatureFlagEnableEnhancedScopes = "EnableEnhancedScopes";
         public const string FeatureFlagDisableDevInDebug = "DisableDevModeInDebug";
         public const string FeatureFlagEnableDiagnosticEventLogging = "EnableDiagnosticEventLogging";
+        public const string FeatureFlagDisableMergedWebHostScriptHostConfiguration = "DisableMergedConfiguration";
```

Comment:
Can we also create a GitHub issue to remove this in a future release?

Reply:
https://github.com/Azure/azure-functions-host/issues/7483
Created issue.